forest/jasmine/traj2stats.py

```diff
@@ -58,6 +63,90 @@ def transform_point_to_circle(lat: float, lon: float, radius: int
     return transform(aeqd_to_wgs84, buffer)
 
 
+def get_nearby_locations(traj: np.ndarray
+                         ) -> Tuple[dict, dict, dict]:
+    """This function returns a dictionary of nearby locations,
+    a dictionary of nearby locations' names, and a dictionary of
+    nearby locations' coordinates.
+
+    Args:
+        traj: numpy array, trajectory
+    Returns:
+        nearby_locations: dictionary, nearby locations
+        nearby_locations_name: dictionary, nearby locations' names
+        nearby_locations_coord: dictionary, nearby locations' coordinates
+    Raises:
+        RuntimeError: if the query to Overpass API fails
+    """
+
+    pause_vec = traj[traj[:, 0] == 2]
+    lat: List[float] = []
+    lon: List[float] = []
```

Comment:
I'd name these variables something like `latitudes` and `longitudes` for readability since they are lists.